.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-shebang-scripts-are-executable
+    -   id: check-executables-have-shebangs
+    -   id: check-merge-conflict
+-   repo: https://github.com/dnephin/pre-commit-golang
+    rev: v0.5.1
+    hooks:
```

Comment:
go-vet and go-fmt are pretty standard pre-commit hooks in Go, as they're built into the language itself and come out of the box. Maybe we should consider adding them to the list?

Reply:
Yea I think so too. I commented about that on the phab task earlier